RazorGenerator.Testing/WebViewPageExtensions.cs

```diff
@@ -158,21 +167,41 @@ private static void EnsureDummyViewEngineRegistered()
             }
         }
         
-        class DummyViewEngine : IViewEngine
+        public class DummyViewEngine : IViewEngine
```

Comment:
Yes. It's public so it can be added along with other test engines. Since it clears engines if it's not in them. Not sure if there's a better way to allow more engines. Callout from EnsureDummyViewEngineRegistered seems a bit too tightly coupled?